Sources/TrustlySDK/TrustlyView.swift

```diff
@@ -503,7 +504,7 @@ public class TrustlyView : UIView, TrustlyProtocol, WKNavigationDelegate, WKScri
         }
 
         if ("local" == env), let urlLocal = localUrl {
-            url = "http://"+urlLocal+"/start/selectBank/"+fn+"?v="+build+"-ios-sdk"
+            url = "http://"+urlLocal+":8000/start/selectBank/"+fn+"?v="+build+"-ios-sdk"
```

Comment:
The adjustment that you mentioned was merged in DEV-163108-spike-passkey-with-backend-swiftui branch, and this branch is no longer used.

The localUrl property can not be deprecated because the older versions of Trustly Alpha can be broken.

Reply:
Which old Trustly Alpha versions concern you?

This application is internal, as is the use of `localUrl`. Since DEV-231710 (merged in May), when Local & Dynamic environments were added to the Trustly Alpha custom use case, `envHost` is the proper parameter.

If `DEV-163108-spike-passkey-with-backend-swiftui` is obsolete, maybe we need to revisit some tickets that were merged to it, because this is not the only feature pushed there in the last months.

Reply:
Renato, we are not talking about Local & Dynamic changes, what we are trying to do here is just allow that SDK and Trustly Alpha works properly to be able to QA validate the core changes in this task (https://trustly.atlassian.net/browse/DEV-236314).

envHost is not a proper parameter for now, we don't have any documentation about this change to guiide the user to change the locaUrl to envHost.